src/client/Microsoft.Identity.Client/ApiConfig/AbstractConfidentialClientAcquireTokenParameterBuilder.cs

```diff
@@ -86,5 +87,23 @@ public T WithProofOfPossession(PoPAuthenticationConfiguration popAuthenticationC
 
             return this as T;
         }
+
+        /// <summary>
+        /// Sends a token request over an MTLS (Mutual TLS) connection using the provided client certificate.
+        /// This method is public and part of the S2S (server-to-server) token binding process, 
+        /// allowing external clients to securely establish a mutual TLS connection.
+        /// </summary>
+        /// <remarks>
+        /// This method should be used when setting up a secure token exchange via MTLS.
+        /// It's important to ensure that the provided X509Certificate2 is valid and secure.
+        /// Note that only the /token request will be transmitted over this MTLS connection.
+        /// </remarks>
+        /// <param name="certificate">An X509Certificate2 object representing the client certificate for MTLS.</param>
+        /// <returns>An instance of the class, allowing for method chaining in a fluent interface style.</returns>
+        public T WithMtlsCertificate(X509Certificate2 certificate)
```

Comment:
Why is this a public API? The connection between the SDK and the STS in handled internally.

Reply:
We discussed offline to remove MTLS POP from this PR.